cmd/argo/commands/clustertemplate/create.go

```diff
@@ -25,6 +25,19 @@ func NewCreateCommand() *cobra.Command {
 	command := &cobra.Command{
 		Use:   "create FILE1 FILE2...",
 		Short: "create a cluster workflow template",
+		Example: `# Create a Cluster Workflow Template with Default Options:
+		argo cluster-template create FILE1.yaml
+
+		# Create a Cluster Workflow Template in JSON Format:
+		argo cluster-template create FILE1.yaml --output json 
+
+		# Create a Cluster Workflow Template with Relaxed Validation (default true):
+		argo cluster-template create FILE1.yaml --strict=false
+
+		# Create a Cluster Workflow Template and Save Output to a File (One of: name|json|yaml|wide):
+		argo cluster-template create FILE1.yaml -o my-output.json
```

Comment:
I actually haven't ran this before, so I would like to test that this works per your example, as the code takes in a format, not a filename

Reply:
@agilgur5  I apologize for my mistake. The --output flag is intended for specifying the output format (e.g., "json" or "yaml") and not for specifying the output file.